modules/ob-about-buildah.adoc

```diff
@@ -1,10 +1,37 @@
 // This module is included in the following assembly:
 //
-// * installing/installing-sample-build-strategies.adoc
+// * about/build-strategies.adoc
 
-:_content-type: CONCEPT
+:_content-type: REFERENCE
 [id="about-buildah_{context}"]
 = Buildah
 
 The `buildah` cluster build strategy uses a Dockerfile to build a container image and pushes it to the target registry. You must specify the Dockerfile in the `spec.paramValues` field of the `Build` CR.
 
+You can share the `buildah` strategy across different namespaces within your cluster because the {builds-operator} installs the `buildah` strategy at cluster level. 
+
+You can configure the following parameters for the `buildah` strategy:
+
+.Configuration parameters for `buildah`
+[options="header"]
+|===
+
+| Name | Type | Description | Default
+
+| `build-args` | array | Key-value pair of the arguments required by the Dockerfile that is used during the build | []
+
+| `registries-block` | array | List of registries that must be blocked | []
+
+| `registries-insecure` | array | List of insecure registries with their fully qualified domain name (FQDN) | []
+
+| `registries-search` | array | List of registries to search short name images | ["registry.redhat.io", "quay.io"]
+
+| `dockerfile` | string | Path of the Dockerfile that is used during the build | "Dockerfile"
+
+| `storage-driver` | string | Storage drivers that are used by `buildah`, such as overlay or vfs | "vfs"
+|===
+
+[NOTE]
+====
+For more details, see _Configuring build strategies_ in the _Additional resources_ section.
```

Comment:
```suggestion
For more information, see _Configuring build strategies_ in the _Additional resources_ section.
```
I know it is a nit, but our guidelines require "information" [1]
[1] https://redhat-documentation.github.io/supplementary-style-guide/#cross-references